cpp/ps0001/ps0001_sol.cpp

```diff
@@ -4,15 +4,19 @@ using namespace std;
 
 int main()
 {
-    int a ,b ,c ;
-    int s ;
-    cout<< "Enter first number \n" ;
-    cin>> a;
-    cout<< "Enter second number \n" ;
-    cin>> b ;
-     cout<< "Enter third number \n" ;
-     cin>> c ;
-     s= a + b + c ;
-     cout<< " The sum is "<< s << endl ;
+    // prompting users for 3 numbers to add 1/2
+    unsigned short a, b, c;
+    unsigned short s;
+    cout << "Enter first number \n";
+    cin >> a;
+    cout << "Enter second number \n";
+    cin >> b;
+    cout << "Enter third number \n";
+    cin >> c;
+    
+    // outputting the result of the summation process 3/3
+    s = a + b + c ;
+    cout << "The sum is " << s << endl;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as program is user-friendly